src/Reporter/Reporter.php

```diff
@@ -35,21 +35,21 @@ public function __construct()
 
     public function progress($current, $total)
     {
-        echo sprintf("Reviewing file %d of %d.\r", $current, $total);
+        echo sprintf("Reviewing %d of %d." . PHP_EOL, $current, $total);
```

Comment:
It doesn't clear the line, except on Windows. That's why I replaced the `\r` with `PHP_EOL`.